docs/kubernetes/mailu/ingress.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
This does not work. It should be `networking.k8s.io/v1beta1`

Reply:
Can you confirm that this works on 1.16? extensions/v1beta1 still works for me and it's going to finally change in 1.20 (https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/).

But apps/v1 is definitely wrong. I must have overlooked this when reviewing the change.

Reply:
yeah I've just upgraded to 1.16 and its working